scripts/fetch-ui-assets.sh

```diff
@@ -18,7 +18,7 @@ declare -r SCRIPT_DIR=$(cd $(dirname ${0}) >/dev/null 2>&1 && pwd)
 declare -r ROOT_DIR=$(dirname ${SCRIPT_DIR})
 declare -r STATIC_DIR="$ROOT_DIR/static"
 
-UI_RELEASE="OSS-v2.6.0"
+UI_RELEASE="OSS-v2.6.1"
```

Comment:
Should this be "OSS-Master" or should the comment above  on line 8 change?

Reply:
I changed the comment to reflect what the actual procedure should be.